### DISCUSSION TITLE
Users not shown full list of models

### DISCUSSION THREAD

**Discussion by Shawneau** [open-webui/open-webui#818] [Q&A] [answered]
As admin I have 6 models available but my users only see 5, anything I need to do to get them to see all my latest models? Thanks

**Comment by justinh-rahb** [comment]
That's odd... there's currently no mechanism in the WebUI to even limit access by user/group to different models or APIs (though this is something we'd like).

**Comment by Shawneau** [comment, marked as answer]
User cleared cache and now it's there...

**Reply by justinh-rahb** [reply]
There's two intractable problems in computer science:
1. naming things
2. cache invalidation
3. off-by-one errors.
